README.md

```diff
@@ -386,13 +386,13 @@ await octokit.request("POST /repos/{owner}/{repo}/issues", {
 });
 ```
 
-`octokit.request` can be used to send requests to other domains by passing a full URL. It can also be used to send requests to endpoints that are not (yet) documented on [GitHub's REST API documentation](https://docs.github.com/rest).
+In both cases a given request is authenticated, retried, and throttled transparently by the `octokit` instance which also manages the `accept` and `user-agent` headers as needed.
 
-#### Endpoint methods
+`octokit.request` is somewhat lower level and can be used to send requests to other domains by passing a full URL and to send requests to endpoints that are not (yet) documented in [GitHub's REST API documentation](https://docs.github.com/rest).
 
-**plugin**: [`@octokit/plugin-rest-endpoint-methods`](https://github.com/octokit/plugin-rest-endpoint-methods.js/#readme).
+#### `octokit.rest` endpoint methods ([`@octokit/plugin-rest-endpoint-methods`](https://github.com/octokit/plugin-rest-endpoint-methods.js/#readme))
```

Comment:
This will mess with the anchors so you may not want to use this exact form. could just leave it in the text as a link in "See the xxx doc for more details". Or consider making the section heading text be the link text. Depends if you want some formal exposure of the rest/request/graphql component names